student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, parent_permision, name = 'unknown')
```

Comment:
- [ ] Kindly make sure to give value to parent permission. Like `parent_permission = true`